src/components/PaymentMethodForm/PaymentMethodFormModalRenderer/PaymentMethodFormModalRenderer.tsx

```diff
@@ -20,23 +22,23 @@ const PaymentMethodFormModalRenderer = ({
     isCreateSuccessful,
     isUpdateSuccessful,
     onResetFormState,
+    resetCreate,
+    resetUpdate,
     setIsError,
     updateError,
 }: TPaymentMethodFormModalRendererProps) => {
     const { localize } = useTranslations();
     const { hideModal, isModalOpenFor, showModal } = useModalManager();
 
     useEffect(() => {
-        if (
+        if (createError || updateError) {
+            showModal('PaymentMethodErrorModal');
```

Comment:
when this is thrown, shouldn't this hide the buy/sell form?

Reply:
you mean the stacking of modals issue or actually hide the modal?

Reply:
i dont think it should hide it, cause in prod, if we get any error, it just closes the error modal and the user can go back to the buy sell form

Reply:
yeah.. not hiding..  i meant the stacking.. in the current production, when the error is thrown, the one behind is not visible.

Reply:
yeah i saw it happening, ill see if i can change it